packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/get-prometheus-exporter-metrics/get-prometheus-exporter-metrics-v1.ts

```diff
@@ -0,0 +1,91 @@
+import { Express, Request, Response } from "express";
+
+import {
+  Logger,
+  LoggerProvider,
+  LogLevelDesc,
+  Checks,
+} from "@hyperledger/cactus-common";
+
+import {
+  IWebServiceEndpoint,
+  IExpressRequestHandler,
+} from "@hyperledger/cactus-core-api";
+
+import OAS from "../../json/openapi.json";
+
+import { registerWebServiceEndpoint } from "@hyperledger/cactus-core";
+
+import { PluginLedgerConnectorFabric } from "../plugin-ledger-connector-fabric";
+import { PrometheusExporter } from "../prometheus-exporter/prometheus-exporter";
+
+export interface IGetPrometheusExporterMetricsV1Options {
+  logLevel?: LogLevelDesc;
+  connector: PluginLedgerConnectorFabric;
+}
+
+export class GetPrometheusExporterMetricsV1 implements IWebServiceEndpoint {
```

Comment:
@jagpreetsinghsasan Could you add the `Endpoint` suffix to the class and the options interface names? I know it makes it super long but hopefully keeping with the convention will pay dividends on the long run.

Reply:
Fixed